serpentTools/settings/messages.py

```diff
@@ -0,0 +1,84 @@
+"""
```

Comment:
I think this file has been converted to DOS line endings, `\r\n`. You can change this in pycharm  and run `dos2unix` to convert back to normal line endings.
Or you can use git to revert this file to the version on `develop`